apps/examples/esp32_wifi_demo/esp32_tash_main.c

```diff
@@ -0,0 +1,100 @@
+/******************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <apps/shell/tash.h>
+#include <errno.h>
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define ESP32_TASH_PRI      (100)
+#define ESP32_TASH_STAKSIZE (4096)
```

Comment:
I think those thread params are for WiFi demo thread after tash cmd invoked
How about we rename those "*_TASH_*".

Reply:
ESP32_TASH_PRI-> ESP32_WIFI_PRI
ESP32_TASH_STAKSIZE -> ESP32_WIFI_STACKSIZE